elements/bulbs-pollzone-map/bulbs-pollzone-map.js

```diff
@@ -106,7 +106,7 @@ class BulbsPollzoneMap extends BulbsHTMLElement {
         let percent = Math.round(numVotes / totalVotes * 100);
 
         return `<li class='result'>
-          <div class='bar' style='background-color: ${this.fillKey(question.id)}; width: ${percent}%;'></div>
+          <div class='bar' style='background-color: ${this.fillKey(question['id'])}; width: ${percent}%;'></div>
```

Comment:
this was just linting - actually removed from this PR (and merged earlier https://github.com/theonion/bulbs-elements/pull/238)